config/Config-kernel.in

```diff
@@ -313,6 +313,21 @@ config KERNEL_DEBUG_INFO
 	help
 	  This will compile your kernel and modules with debug information.
 
+config KERNEL_DEBUG_INFO_BTF
+	bool "Enable additional BTF type information"
+	default n
+	depends on KERNEL_DEBUG_INFO
+	depends on !LINUX_4_14 && !LINUX_4_19
```

Comment:
There is only 5.4 in master, so this line needs to be dropped.

Reply:
Yes, when development started 5.4 was still in progress. Will remove now.